src/components/WebComponent/WebComponentLoader/WebComponentLoader.js

```diff
@@ -15,12 +15,11 @@ const ProjectComponentLoader = (props) => {
     }
     dispatch(setSenseHatAlwaysEnabled(typeof sense_hat_always_enabled !== 'undefined'))
     dispatch(setProject(proj))
-    dispatch(setProjectLoaded(true))
   }, []);
```

Comment:
Feel like if we are changing this file, we could fix up the useEffect array as per linting